Translations update from Weblate

SirClawsALot opened the pull request:
Translations update from [Weblate](https://weblate.flawcra.cc) for [Syrup/Content](https://weblate.flawcra.cc/projects/syrup/content/).



Current translation status:

![Weblate translation status](https://weblate.flawcra.cc/widget/syrup/content/horizontal-auto.svg)

Commit:
Translated using Weblate (German)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/de/

Commit:
Translated using Weblate (German)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/de/

Commit:
Translated using Weblate (French (Canada))

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/fr_CA/

Commit:
Translated using Weblate (Turkish)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/tr/

Commit:
Translated using Weblate (Romanian)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/ro/

Commit:
Translated using Weblate (Czech)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/cs/

Commit:
Translated using Weblate (Danish)

Currently translated at 90.9% (20 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/da/

ImGajeed76 commented on Extension-React/public/_locales/de/messages.json:
```diff
@@ -24,7 +24,7 @@
     },
     "testing_coupons": {
-        "message": "Gutscheine testen..."
+        "message": "Coupons testen..."
```
As a German, I think that it should either be "coupons testen" or if you really want the German word for it "Rabattcodes testen" but not "Gutscheine"

0xseal replied:
agreed

ImGajeed76 commented on Extension-React/public/_locales/de/messages.json:
```diff
@@ -24,7 +24,7 @@
+        "message": "Coupons testen..."
     },
     "testing_coupon_current_of_total": {
         "message": "Teste Gutschein %currentIndex% von %total%"
```
Same thing here.

ImGajeed76 commented on Extension-React/public/_locales/de/messages.json:
```diff
@@ -61,5 +61,8 @@
         "message": "Thema"
+    },
+    "language_name": {
+        "message": "English"
```
And "English" in german is "Englisch"

0xseal replied:
+1, im still not sure where language_name is even used though, as I thought this was the name of the language not the translation of "English"

ImGajeed76 replied:
Yes, i was confused as well.

Commit:
Added translation using Weblate (Norwegian Nynorsk)

Commit:
Added translation using Weblate (Norwegian Bokmål)

Commit:
Translated using Weblate (Norwegian Nynorsk)

Currently translated at 95.4% (21 of 22 strings)

Translation: Syrup/Content
Translate-URL: https://weblate.flawcra.cc/projects/syrup/content/nn/

Abstra208 commented:
I'll close this...
